<launch>
	<node pkg="openfabmap2" type="learn_node" name="learn_node" output="screen">
		<remap from="image" to="/camera/rgb/image_raw"/>
		<param name="vocab" value="$(find openfabmap2)/codebooks/new/vocab.yml"/>
		<param name="clTree" value="$(find openfabmap2)/codebooks/new/clTree.yml"/>
		<param name="trainbows" value="$(find openfabmap2)/codebooks/new/trainbows.yml"/>
		<param name="DetectorType" value="BRAND"/>
		<param name="DescriptorType" value="BRAND"/>
		<param name="sampleRate" value="0.65"/>
		<param name="maxImages" value="10"/>
		<param name="clusterSize" value="64"/>
		<param name="visualise" value="true"/>
	</node>
</launch>
